Bug: Api Dockerfile fails to build

`docker-compose up --build` fails with this error log:

```
#16 127.8 error /srv/gauzy/node_modules/sharp: Command failed.
#16 127.8 Exit code: 1
#16 127.8 Command: (node install/libvips && node install/dll-copy && prebuild-install --runtime=napi) || (node-gyp rebuild && node install/dll-copy)
#16 127.8 Arguments: 
#16 127.8 Directory: /srv/gauzy/node_modules/sharp
#16 127.8 Output:
#16 127.8 info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.9.1/libvips-8.9.1-linuxmusl-x64.tar.gz
#16 127.8 ERR! sharp read ECONNRESET
#16 127.8 info sharp Attempting to build from source via node-gyp but this may fail due to the above error
#16 127.8 info sharp Please see https://sharp.pixelplumbing.com/install for required dependencies
#16 127.8 gyp info it worked if it ends with ok
#16 127.8 gyp info using node-gyp@6.1.0
#16 127.8 gyp info using node@14.5.0 | linux | x64
#16 127.8 gyp info find Python using Python version 2.7.18 found at "/usr/bin/python"
#16 127.8 gyp http GET https://unofficial-builds.nodejs.org/download/release/v14.5.0/node-v14.5.0-headers.tar.gz
#16 127.8 gyp http 200 https://unofficial-builds.nodejs.org/download/release/v14.5.0/node-v14.5.0-headers.tar.gz
#16 127.8 gyp http GET https://unofficial-builds.nodejs.org/download/release/v14.5.0/SHASUMS256.txt
#16 127.8 gyp http 200 https://unofficial-builds.nodejs.org/download/release/v14.5.0/SHASUMS256.txt
#16 127.8 gyp info spawn /usr/bin/python
#16 127.8 gyp info spawn args [
#16 127.8 gyp info spawn args   '/srv/gauzy/node_modules/electron-rebuild/node_modules/node-gyp/gyp/gyp_main.py',
#16 127.8 gyp info spawn args   'binding.gyp',
#16 127.8 gyp info spawn args   '-f',
#16 127.8 gyp info spawn args   'make',
#16 127.8 gyp info spawn args   '-I',
#16 127.8 gyp info spawn args   '/srv/gauzy/node_modules/sharp/build/config.gypi',
#16 127.8 gyp info spawn args   '-I',
#16 127.8 gyp info spawn args   '/srv/gauzy/node_modules/electron-rebuild/node_modules/node-gyp/addon.gypi',
#16 127.8 gyp info spawn args   '-I',
#16 127.8 gyp info spawn args   '/home/node/.cache/node-gyp/14.5.0/include/node/common.gypi',
#16 127.8 gyp info spawn args   '-Dlibrary=shared_library',
#16 127.8 gyp info spawn args   '-Dvisibility=default',
#16 127.8 gyp info spawn args   '-Dnode_root_dir=/home/node/.cache/node-gyp/14.5.0',
#16 127.8 gyp info spawn args   '-Dnode_gyp_dir=/srv/gauzy/node_modules/electron-rebuild/node_modules/node-gyp',
#16 127.8 gyp info spawn args   '-Dnode_lib_file=/home/node/.cache/node-gyp/14.5.0/<(target_arch)/node.lib',
#16 127.8 gyp info spawn args   '-Dmodule_root_dir=/srv/gauzy/node_modules/sharp',
#16 127.8 gyp info spawn args   '-Dnode_engine=v8',
#16 127.8 gyp info spawn args   '--depth=.',
#16 127.8 gyp info spawn args   '--no-parallel',
#16 127.8 gyp info spawn args   '--generator-output',
#16 127.8 gyp info spawn args   'build',
#16 127.8 gyp info spawn args   '-Goutput_dir=.'
#16 127.8 gyp info spawn args ]
#16 127.8 gyp info spawn make
#16 127.8 gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
#16 127.8 make: Entering directory '/srv/gauzy/node_modules/sharp/build'
#16 127.8   TOUCH Release/obj.target/libvips-cpp.stamp
#16 127.8   CC(target) Release/obj.target/nothing/node_modules/node-addon-api/nothing.o
#16 127.8   AR(target) Release/obj.target/node_modules/node-addon-api/nothing.a
#16 127.8   COPY Release/nothing.a
#16 127.8   CXX(target) Release/obj.target/sharp/src/common.o
#16 127.8 ../src/common.cc:23:10: fatal error: vips/vips8: No such file or directory
#16 127.8    23 | #include <vips/vips8>
#16 127.8       |          ^~~~~~~~~~~~
#16 127.8 compilation terminated.
#16 127.8 make: *** [sharp.target.mk:139: Release/obj.target/sharp/src/common.o] Error 1
#16 127.8 make: Leaving directory '/srv/gauzy/node_modules/sharp/build'
#16 127.8 gyp ERR! build error 
#16 127.8 gyp ERR! stack Error: `make` failed with exit code: 2
#16 127.8 gyp ERR! stack     at ChildProcess.onExit (/srv/gauzy/node_modules/electron-rebuild/node_modules/node-gyp/lib/build.js:194:23)
#16 127.8 gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
#16 127.8 gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
#16 127.8 gyp ERR! System Linux 5.7.8-arch1-1
#16 127.8 gyp ERR! command "/usr/local/bin/node" "/srv/gauzy/node_modules/.bin/node-gyp" "rebuild"
#16 127.8 gyp ERR! cwd /srv/gauzy/node_modules/sharp
#16 127.8 gyp ERR! node -v v14.5.0
#16 127.8 gyp ERR! node-gyp -v v6.1.0
#16 127.8 gyp ERR! not ok
#16 127.8 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

This seems to be because of missing header files for compiling. See also: https://github.com/lovell/sharp/issues/1882

Related issue:

```
api          | internal/modules/cjs/loader.js:1033
api          |   throw err;
api          |   ^
api          | 
api          | Error: Cannot find module 'passport-azure-ad-oauth2'
api          | Require stack:
api          | - /srv/gauzy/main.js
api          |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1030:15)
api          |     at Function.Module._load (internal/modules/cjs/loader.js:899:27)
api          |     at Module.require (internal/modules/cjs/loader.js:1090:19)
api          |     at require (internal/modules/cjs/helpers.js:75:18)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:25625:18)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:25600:36)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:5622:30)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:24561:23)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:19979:34)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30)
api          |     at Object.<anonymous> (/srv/gauzy/main.js:19930:22)
api          |     at __webpack_require__ (/srv/gauzy/main.js:20:30) {
api          |   code: 'MODULE_NOT_FOUND',
api          |   requireStack: [ '/srv/gauzy/main.js' ]
api          | }
```
Could you please take a look @evereq ?

--- a/.deploy/api/Dockerfile
+++ b/.deploy/api/Dockerfile
@@ -2,7 +2,7 @@
 FROM node:alpine AS dependencies
 
 RUN apk --update add bash && \
-    apk add --no-cache dos2unix g++ git make python
+    apk add --no-cache dos2unix g++ git make python vips-dev
 
 RUN mkdir /srv/gauzy && chown -R node:node /srv/gauzy
 

